src/devices/SensorHub/SensorHub.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.SensorHub
+{
+    /// <summary>
+    /// SensorHub
+    /// </summary>
+    public class SensorHub
+    {
+        /// <summary>
+        /// Default I2C bus id
+        /// </summary>
+        public const int DefaultI2cBusId = 1;
+
+        /// <summary>
+        /// Default I2C address
+        /// </summary>
+        public const int DefaultI2cAddress = 0x17;
+
+        private readonly I2cDevice _i2cDevice;
+
+        /// <summary>
+        /// Creates a SensorHub with the default bus and connection values
+        /// </summary>
+        /// <returns>SensorHub</returns>
+        public static SensorHub Create()
+        {
+            return new(I2cDevice.Create(new I2cConnectionSettings(DefaultI2cBusId, DefaultI2cAddress)));
+        }
```

Comment:
I2cDevice.Create() should not be called from bindings directly. We're adding a new abstraction layer that hides these hardware-dependent factory methods from the user. So better remove this method.

Reply:
removed